zerver/views/user_settings.py

```diff
@@ -50,6 +50,9 @@ def confirm_email_change(request, confirmation_key):
 
         context = {'realm': obj.realm,
                    'new_email': new_email,
+                    'unsub_welcome' : notifications.one_click_unsubscribe_link(user_profile,"welcome"),
+                    'unsub_digest' : notifications.one_click_unsubscribe_link(user_profile,"digest"),
+                    'unsub_missed' : notifications.one_click_unsubscribe_link(user_profile,"missed messages"),
```

Comment:
We usually write this as `from zerver.lib.notifications import one_click_unsuscribe_link` at the top so we don't need to write all the `notifications.`s here (it's not super-necessary to know where the `one_click_unsubscribe_link` is coming from)